.github/workflows/python-tests.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           pipenv run flake8 .
       - name: Run Type Checker
         run: |
-          pipenv run mypy src/build_workflow tests/tests_build_workflow src/checkout_workflow tests/tests_checkout_workflow src/run_assemble.py tests/test_run_assemble.py src/assemble_workflow tests/tests_assemble_workflow src/manifests tests/tests_manifests src/paths tests/tests_paths src/system tests/tests_system src/ci_workflow tests/tests_ci_workflow src/manifests_workflow tests/tests_manifests_workflow
+          pipenv run mypy .
```

Comment:
In my exp, using `.` will not properly check all the relationships between the types.

Reply:
Can you please elaborate? Regardless of how I run it I get the same results, 16.07% imprecise, 18241 LOC.

Reply:
I think previously I have to specify all the folders for the mypy to check errors, or it will skip or pass even though I have wrong type.

Reply:
I am not seeing this, but I could be wrong. How do I know? Running `.` vs. `src tests` makes no difference. AFAIK the reason why we have this list here is that we didn't have typings in all files.

Reply:
I can probably be wrong here so I was wondering if you can confirm the usages, but if not I am ok we are using `.` for now.